src/Xamarin.Android.Build.Tasks/Tasks/InstallApkSet.cs

```diff
@@ -28,8 +30,11 @@ internal override CommandLineBuilder GetCommandLineBuilder ()
 			// --modules: List of modules to be installed, or "_ALL_" for all modules.
 			// Defaults to modules installed during first install, i.e. not on-demand.
 			// Xamarin.Android won't support on-demand modules yet.
-			cmd.AppendSwitchIfNotNull ("--modules ", "_ALL_");
-			
+			if (Modules == null)
+				cmd.AppendSwitchIfNotNull ("--modules ", "_ALL_");
+			else
+				cmd.AppendSwitchIfNotNull ("--modules", string.Join (",", Modules));
```

Comment:
Do we need to put quotes if the paths have a space in them?

Reply:
Probably lol